util/deploy-production.sh

```diff
@@ -7,27 +7,25 @@
 set -e
 
 usage() {
-  USAGE="Usage: deploy-production.sh [-f] [-b]
+  USAGE="Usage: deploy-production.sh [-f] [-b] [-q]
     -b : skip GitHub issue creation
-    -f : Always deploy (even if checks have failed)"
+    -f : Always deploy (even if checks have failed)
+    -q : Disable all interactive prompts when running gcloud deploy commands"
```

Comment:
Nit: the quiet flag removes more things in deploy.sh, so I would rephrase it perhaps as "Disable all interactive prompts and debugging output".

While you are here, do you mind making capitalization more uniform by changing "skip" to "Skip" in the line above?

Reply:
Done!